CMakeLists.txt

```diff
@@ -40,15 +40,6 @@ set(LLVM_ENABLE_TERMINFO OFF CACHE BOOL "")
 # LLVM settings that have an effect on the MLIR dialect
 set(LLVM_TARGETS_TO_BUILD "X86;AMDGPU" CACHE STRING "")
 
-# Build the ROCm conversions and run according tests if the AMDGPU backend
```

Comment:
The logic is placed at `external/llvm-project/mlir/CMakeLists.txt` too so we don't need duplication here.